cmd/gitbase/command/server.go

```diff
@@ -39,16 +44,24 @@ type Server struct {
 	PilosaURL     string   `long:"pilosa" default:"http://localhost:10101" description:"URL to your pilosa server" env:"PILOSA_ENDPOINT"`
 	IndexDir      string   `short:"i" long:"index" default:"/var/lib/gitbase/index" description:"Directory where the gitbase indexes information will be persisted." env:"GITBASE_INDEX_DIR"`
 	DisableSquash bool     `long:"no-squash" description:"Disables the table squashing."`
-	// IgnoreGitErrors by default when gitbase encounters and error in a
-	// repository it stops the query. With this parameter it won't complain and
-	// just skip those rows or repositories.
+	TraceEnabled  bool     `long:"trace" env:"GITBASE_TRACE" description:"Enables jaeger tracing"`
```

Comment:
You have a default env variable to do this: `JAEGER_DISABLED`. I don't know the default value. We should check that. Maybe the default tracer with no env variables is a `noop tracer`

Reply:
By default it is enabled and the tracer is jaeger. That's what happens now when you specify `--tracer` and no env vars. I preferred tracing to be completely disabled by default.